src/services/middlewareServices/AuthenticationMiddlewareService.ts

```diff
@@ -1,27 +1,34 @@
 // Import logger
-const logger = require("@logger/logger")
+import logger from "@logger/logger"
 
 // Import errors
-const { AuthError } = require("@errors/AuthError")
-const { NotFoundError } = require("@errors/NotFoundError")
+import { AuthError } from "@errors/AuthError"
 
-const jwtUtils = require("@utils/jwt-utils")
+import jwtUtils from "@utils/jwt-utils"
 
-const { BadRequestError } = require("@root/errors/BadRequestError")
-const { sitesService } = require("@services/identity")
+import { E2E_ISOMER_ID } from "@root/constants"
+import { BadRequestError } from "@root/errors/BadRequestError"
 
 const { E2E_TEST_REPO, E2E_TEST_SECRET, E2E_TEST_GH_TOKEN } = process.env
 const E2E_TEST_USER = "e2e-test"
-const E2E_ISOMER_ID = "e2e-id"
 const E2E_TEST_EMAIL = "test@e2e"
-const GENERAL_ACCESS_PATHS = ["/v1/sites", "/v1/auth/whoami"]
-
-class AuthMiddlewareService {
-  constructor({ identityAuthService }) {
-    this.identityAuthService = identityAuthService
-  }
-
-  verifyE2E({ cookies, url }) {
+const GENERAL_ACCESS_PATHS = [
+  "/v1/sites",
+  "/v1/auth/whoami",
+  "/v2/sites",
+  "/v2/auth/whoami",
+]
+
+export default class AuthenticationMiddlewareService {
+  verifyE2E({
+    cookies,
+    url,
+  }: {
+    cookies: {
+      isomercmsE2E?: string
+    }
+    url: string
```

Comment:
could we define this above for readability? usually 1-2 lines is ok for inlining type-defs but this is getting slightly long

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/479/commits/55ed491e445f3f25993036851c1417bf95f18a96!